packages/flutter/test/widgets/text_test.dart

```diff
@@ -25,13 +25,58 @@ void main() {
     text = tester.firstWidget(find.byType(RichText));
     expect(text, isNotNull);
     expect(text.textScaleFactor, 1.0);
+  });
+
+  testWidgets('Text respects textScaleFactor with default font size',
+      (WidgetTester tester) async {
+    await tester.pumpWidget(
+        const Center(child: const Text('Hello', textDirection: TextDirection.ltr)));
```

Comment:
use flutter style (if there's a line break after a (, there should be a line break before the ); also, two-space indent